google/resource_bigtable_instance.go

```diff
@@ -265,3 +312,77 @@ func expandBigtableClusters(clusters []interface{}, instanceID string) []bigtabl
 	}
 	return results
 }
+
+func resourceBigtableInstanceClusterReorderTypeList(diff *schema.ResourceDiff, meta interface{}) error {
+	keys := diff.GetChangedKeysPrefix("cluster")
```

Comment:
I'd imagine we can omit this check and run through this logic every time, unless you have a reason otherwise?

Reply:
yeah good call, i think i was initially just copying the customize diff example you linked